src/Servers/Kestrel/Transport.Sockets/src/SocketTransportOptions.cs

```diff
@@ -77,6 +94,9 @@ public class SocketTransportOptions
     /// calls <see cref="Socket.Bind"/> as part of its implementation, so implementors
     /// using this method do not need to call it again.
     /// </remarks>
+    /// <remarks>
+    /// Defaults to <see cref="CreateBoundListenSocket"/>.
```

Comment:
```suggestion
    /// Defaults to <see cref="CreateDefaultBoundListenSocket"/>.
```

I'll make sure main gets this update too, so we avoid merge conflicts.